paper/paper.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'wdl2cwl: A Python tool for converting WDL workflow files to CWL workflows'
+tags:
+  - Python
+  - workflow
+  - workflow description language
+  - common workflow language
+  - Language
+authors:
+  - name: Dinithi Wickramaratne
+  - name: Dennis Chukwunta
+  - name: Michael R. Crusoe
+    orcid: 0000-0002-2961-9670
+  - name: Bruno P. Kinoshita
+    orcid: 0000-0001-8250-4074
+  - name: Peter Amstutz
+    orcid: 0000-0003-3566-7705
+date:  August 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+Computational workflows are used in data analysis, enabling innovation and 
+decision-making in various fields of science. There are several competing workflow description languages, 
+WDL (Workflow description Language) and CWL (Common Workflow Languaage) are two examples
+of popularly used systems. WDL is a way to specify data processiing workflows with 
+a human-readable and writeable syntax. WDL makes it straightforward to define 
+complex analysis tasks, chain them together in workflows, and parallelize their execution [@wdl].
+CWL is a way to describe command line tools and connect 
```

Comment:
Ditto here, about the `is a way`. I think if descriptions are not identical, and point to different features of each workflow definition language, then this sentence would probably start with something like "On the other hand, CWL is...". 

But take my advice with a pinch of salt, as I studied academic writing but didn't have to use it a lot. You can search for sentence starters, or other academic writing pages that list several examples, e.g.: https://www.op.ac.nz/assets/Uploads/f373ebe246/Sentence-starters.pdf